src/DynamoCoreWpf/Rendering/HelixRenderPackage.cs

```diff
@@ -14,7 +14,7 @@ namespace Dynamo.Wpf.Rendering
     /// A Helix-specific IRenderPackageFactory implementation.
     /// </summary>
     [Obsolete("Do not use! This will be moved to a new project in a future version of Dynamo.")]
```

Comment:
also remove the obsolete message I guess?
I think our original plan was to remove these after providing an alternative API - something like a GraphicsSDK package that kept Helix hidden but we never did that - I wonder if this change will break DynamoShape with no workaround.

@LongNguyenP can you look at that?

Reply:
I did not change this in the end.